packages/cfpb-atomic-component/src/utilities/atomic-helpers.js

```diff
@@ -9,7 +9,7 @@
    - Atom
    ========================================================================= */
 
-import * as dataHook from './data-hook';
+import { add, contains, remove } from './data-hook';
```

Comment:
Why import the individual methods separately? For clarity?

Reply:
Yeah so if you search for the methods you will get the import statement in the search results. Also, if data-hook has more added to it, it won't be automatically pulled into this file where it's unneeded.